.github/workflows/clang-tidy.yml

```diff
@@ -48,10 +51,28 @@ jobs:
         env:
           ONEDNN_ACTION: configure
 
-      - name: Build oneDNN
+      - name: Check source files
         run: |
-          .github/automation/x64/build_linters.sh 2>&1 | tee build.log
-          grep -i "warning:" build.log | sort -u
-          grep -q -i "warning:" build.log && exit 1 || true
-        env:
-          ONEDNN_ACTION: build
+          touch source-check.log
+          for file in $(git diff --name-only ${{ github.event.pull_request.base.sha }} | grep -E '\.(c|cpp)$'); 
```

Comment:
I tested this one and it looks like there's no value in running clang-tidy on `.c` files.